lab-steven/readme.md

```diff
@@ -0,0 +1 @@
+# Lab 08 - Single Resource API
```

Comment:
why no readme tho? 

basic scaffold: What does your app do, how to npm install, what dependencies are required, what endpoints to hit, and what to pass in.